example/mosmix_forecasts.py

```diff
@@ -36,8 +36,8 @@ def mosmix_example():
     mosmix = DWDMosmixData(
         station_ids=['01028', '01092'],
         parameters=['DD', 'ww'],
-        start_date=ForecastDate.LATEST,  # automatically set if left empty
-        period_type=PeriodType.FORECAST_SHORT,
+        start_date=DWDFcstDate.LATEST,  # automatically set if left empty
+        forecast_type=PeriodType.FORECAST_SHORT,
```

Comment:
Is `forecast_type` better than `period_type`?

Reply:
You're right it should have been forecast_type always as S and L never refered to the forecast range but rather the number of parameters...